openpype/hosts/max/api/lib.py

```diff
@@ -384,12 +365,20 @@ def check_colorspace():
                  "because Max main window can't be found.")
     if int(get_max_version()) >= 2024:
         color_mgr = rt.ColorPipelineMgr
-        if color_mgr.Mode != rt.Name("OCIO_Custom"):
+        project_name = get_current_project_name()
+        project_settings = get_project_settings(
+        project_name)
+        global_imageio = project_settings["global"]["imageio"]
+        max_config_data = colorspace.get_imageio_config(
+        project_name, "max", project_settings)
+        config_enabled = global_imageio["activate_global_color_management"] or (
+            max_config_data)
```

Comment:
I believe the config enabled check isn't needed explicitly because I believe `get_imageio_config` actually returns an empty dict if it is disabled, right?